### PR TITLE
fixing object-reclaimer review comments

### PR DESCRIPTION
**Pull request by jackyalbo**
### Explain the changes
1. 

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

File: src/server/object_services/schemas/object_part_indexes.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = [{
             start: 1,
         },
         options: {
+            name: 'undeleted_parts_by_objects_and_offset_index',
```
Comment (guymguym) [inline src/server/object_services/schemas/object_part_indexes.js]: Can you verify that providing a name to existing index will indeed set its name?

File: src/server/object_services/schemas/object_md_indexes.js [hunk 2]
```diff
@@ -145,4 +144,21 @@ module.exports = [
         }
     },
 
+    //////////////////////
+    // OBJECT RECLAIMER //
+    //////////////////////
+
+    {
+        fields: {
+            deleted: 1,
```
Comment (guymguym) [inline src/server/object_services/schemas/object_md_indexes.js]: I remember there was some issue with having two indexes that have the same fields but differ only in their partialFilterExpression - see here:
https://github.com/noobaa/noobaa-core/blob/f6f10dcac7457c136bf858d0d07d946f54ba057d/src/server/object_services/schemas/object_md_indexes.js#L14-L16
Is that working for you?

File: src/server/object_services/schemas/object_part_indexes.js (hunk 1 above)
Comment (guymguym) [reply src/server/object_services/schemas/object_part_indexes.js]: BTW I don't think we need the name to say "undeleted" explicitly, I would keep it as "parts_by_objects_and_offset_index"

Comment (jackyalbo) [reply src/server/object_services/schemas/object_part_indexes.js]: It doesn't set his name - changed to the default old name 'obj_1_start_1'

File: src/server/object_services/schemas/object_md_indexes.js (hunk 2 above)
Comment (jackyalbo) [reply src/server/object_services/schemas/object_md_indexes.js]: it doesn't added reclaimed to the fields list (will always be null)

File: src/server/object_services/schemas/object_part_indexes.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ module.exports = [{
             start: 1,
         },
         options: {
+            name: 'obj_1_start_1',
```
Comment (guymguym) [inline src/server/object_services/schemas/object_part_indexes.js]: Just add a comment that we use the existing name to avoid recreating it

**Review by guymguym — APPROVED** [review]

**Pull request merged**